gtdynamics/universal_robot/sdf.cpp

```diff
@@ -106,13 +105,34 @@ LinkSharedPtr LinkFromSdf(uint8_t id, const sdf::Link &sdf_link) {
   const auto &I = sdf_link.Inertial().Moi();
   inertia << I(0, 0), I(0, 1), I(0, 2), I(1, 0), I(1, 1), I(1, 2), I(2, 0),
       I(2, 1), I(2, 2);
-  const auto wTl = Pose3FromIgnition(sdf_link.Pose());
+
+  /// Call SemanticPose::Resolve so the pose is resolved to the correct frame
+  /// http://sdformat.org/tutorials?tut=pose_frame_semantics&ver=1.7&cat=specification&
+  // Get non-const Pose3d
+  auto jTl = sdf_link.RawPose();
+  // Update from joint frame to base frame in-place.
+  // Base frame is denoted by "".
+  auto errors = sdf_link.SemanticPose().Resolve(jTl, "");
```

Comment:
I think the use of "Resolve" probably means we're not leaning into the new (typical/desired) SDF organization but I have no time to figure it out again.

Reply:
Yep, pretty much. This would be a future change though, beyond the scope of this current PR and something that needs more effort (which I guess won't happen till the lab reconvenes in the fall).